libraries/AP_Motors/AP_Motors_Class.h

```diff
@@ -53,7 +53,9 @@ class AP_Motors {
         MOTOR_FRAME_TYPE_VTAIL = 4,
         MOTOR_FRAME_TYPE_ATAIL = 5,
         MOTOR_FRAME_TYPE_Y6B = 10,
-        MOTOR_FRAME_TYPE_Y6F = 11 // for FireFlyY6
+        MOTOR_FRAME_TYPE_Y6F = 11, // for FireFlyY6
+        MOTOR_FRAME_TYPE_TAILSITTER = 12, // none vectored tailsiters
```

Comment:
I think maybe we are no longer using MOTOR_FRAME_TYPE_TAILSITTER so we don't need this element 12?